ecsact/entt/wrapper/dynamic.hh

```diff
@@ -126,21 +128,23 @@ template<typename C>
 auto context_get(
 	ecsact_system_execution_context*          context,
 	[[maybe_unused]] ecsact_component_like_id component_id,
-	void*                                     out_component_data
+	void*                                     out_component_data,
+	auto&                                     view
 ) -> void {
 	auto        entity = context->entity;
 	const auto& registry = *context->registry;
 
 	assert(registry.template any_of<C>(entity));
 
-	*static_cast<C*>(out_component_data) = registry.template get<C>(entity);
+	*static_cast<C*>(out_component_data) = view.template get<C>(entity);
 }
 
 template<typename C>
 auto context_update(
 	ecsact_system_execution_context*          context,
 	[[maybe_unused]] ecsact_component_like_id component_id,
-	const void*                               in_component_data
+	const void*                               in_component_data,
+	auto&                                     view
```

Comment:
`view` is unused here? Shouldn't it be used below to get the component and beforechange storage?